Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -191,11 +190,11 @@ def PtypString8(data_value):
     def PtypString(data_value):
         if data_value:
             try:
+                data_value = data_value.decode("utf-16-le", errors="ignore").replace('\x00', '')
```

Comment:
@anara123 From what I saw in `msg_parser` are supposed to use `utf-16-le` and not `ascii` or other things, is there a certain reason we first try to encode it differently, and only if it fails are we using `utf-16-le`?
Because I tried to run it with emails in English and it worked well, and all the tests passed.
thanks.